cloud_environment/deployment/azure/deploy.sh

```
@@ -0,0 +1,110 @@
+az keyvault secret set \
+    --vault-name ${KEYVAULT_NAME} \
+    --name "anthropic-api-key" \
+    --value "${ANTHROPIC_API_KEY}"
+
+echo "Step 9: Creating Container Instance..."
+az container create \
+    --resource-group ${RESOURCE_GROUP} \
+    --name ${CONTAINER_NAME} \
+    --image ${IMAGE_TAG} \
+    --registry-login-server ${ACR_LOGIN_SERVER} \
+    --registry-username $(az acr credential show --name ${ACR_NAME} --query username --output tsv) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)